gannonprudhomme/CSCE-482-KNOW-frontend
# Fix CSS and add basic CSS to knowledge panel

## Description

First, this fixes how CSS is loading and simplifies the process a bit more. Next, it deletes `Sample`, as there are various other components that teach how it works. Last, it moves `KnowledgePanel/` folder up one folder, making it the root of the `components/` folder, and then adds some basic CSS to the main `KnowledgePanel` component.

The CSS I added:
- Adds a border to it
- Adds an 8px padding (adds space between the edge of the component and its interior/content, whereas margin adds space around the exterior of the component)
- Adds `width: fit-content` so that it's only as wide as its children/content need to be. Without this, it would take up the entire screen (equivalent to `width: 100%`)

## How to test

Same as before - just check it out and ensure it works as expected. Likewise, glance at the commits and make sure you get the gist of what they're doing (no need to understand the `webpack.config.js` stuff though).

## Screenshots

![chrome_V7WlNNBniK](https://user-images.githubusercontent.com/7295783/112532498-20b75b00-8d66-11eb-8601-f805acca09a3.png)

## Related tasks
